TYPO3.Neos/Configuration/Settings.yaml

```diff
@@ -485,23 +471,6 @@ TYPO3:
     view:
       title: 'Neos Setup'
 
-  Media:
-    thumbnailPresets:
-      'TYPO3.Neos:Thumbnail':
-        maximumWidth: 250
-        maximumHeight: 250
-      'TYPO3.Neos:Preview':
-        maximumWidth: 1000
-        maximumHeight: 1000
```

Comment:
changing these presets is wrong, they belong to Neos and not the media package as they're used throughout the neos backend and not just for the media browser..

also changing them is a breaking change, which is better to avoid